student.rb

```diff
@@ -4,8 +4,8 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(age, classroom, name = 'unknown')
-    super(age, name)
+  def initialize(classroom)
+    super(name, age, parent_permission: true)
```

Comment:
- [ ] your student class is missing a method that shows the b elongs to side of student to classroom
```
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```